infrastructure/pcluster-manager.yaml

```diff
@@ -237,6 +237,9 @@ Resources:
       ApiId: !Ref ApiGateway
       StageName: $default
       AutoDeploy: True
+      DefaultRouteSettings:
+        ThrottlingBurstLimit: 50 
+        ThrottlingRateLimit: 100
```

Comment:
question: are these limits suitable for our use case? Should they be higher?

Reply:
Since the product is installed on the customer account and used mostly by cluster admins (and not end users) these limits should be good!